source/Makefile

```diff
@@ -406,6 +422,67 @@ Core/Gen/Translation_lzfx_font.%.cpp: $(OUTPUT_DIR)/Core/Gen/translation.files/%
 		--compress-font \
 		$*
 
+#

```

Comment:
To be completely open, I have no idea how you are sane after getting the makefile working for this...
I'm going to take this file's functionality on trust mostly :)